CMakeLists.txt

```diff
@@ -1,15 +1,21 @@
 cmake_minimum_required(VERSION 2.8)
 
+set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
```

Comment:
I would have appended to the list:

```cmake
list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
```

but your version using just `set()` is fine by me.

Reply:
I see. I will keep the set one as is more consistent with the surrounding code and it makes more explicit that such variable was unset before (no previous value)